### PR TITLE
Order page

### PR DESCRIPTION
**Pull request by srishtigautam21**
Add Orders Page
Files to check:
OrderCard.jsx
orderCard.css
PriceContext.js
ProfilePage.jsx
OrderPage.jsx

### CODE REVIEW

File: src/component/Card/orderCard.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.small-horizontal-card {
+  display: flex;
+  border: 1px solid var(--BOX-SHADOW);
+  border-radius: 1rem;
+  margin: 1rem;
+}
+.orders-image {
+  border-bottom-left-radius: 1rem;
+  border-top-left-radius: 1rem;
+  /* width: 14rem; */
```
Comment (ashishkatiyar2404) [inline src/component/Card/orderCard.css]: you can remove comments

File: src/component/Context/PriceContext/PriceContext.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { createContext, useState, useContext } from "react";
```
Comment (ashishkatiyar2404) [inline src/component/Context/PriceContext/PriceContext.js:1]: Appreciation:  Clean Code 🔥

File: src/index.js [hunk 3]
```diff
@@ -22,7 +23,9 @@ ReactDOM.render(
         <CardProvider>
           <WishListProvider>
             <CartProvider>
-              <App />
+              <PriceProvider>
+                <App />
+              </PriceProvider>
             </CartProvider>
           </WishListProvider>
         </CardProvider>
```
Comment (ashishkatiyar2404) [inline src/index.js:31]: Card provider 2 times?

File: src/pages/OrderPage/OrderPage.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { Link } from "react-router-dom";
+import { useCart } from "../../component";
+import { OrdersCard } from "../../component";
```
Comment (ashishkatiyar2404) [inline src/pages/OrderPage/OrderPage.jsx]:
```suggestion
import { useCart,OrdersCard } from "../../component";

```

File: src/pages/OrderPage/OrderPage.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    <div className='profile-page'>
+      <div className='price-details-container'>
+        <h3>Account Details</h3>
+        <div className='divider'></div>
+        <Link className='link link-color' to='/profile'>
+          Profile
+        </Link>
+
+        {/* <div className='divider'></div>
+        <p>Address</p> */}
```
Comment (ashishkatiyar2404) [inline src/pages/OrderPage/OrderPage.jsx]: you can remove comments

**Review by ashishkatiyar2404 — APPROVED** [review]
Suggested some changes pls look into it.
Your code is clean and looks good to me. You can merge💯

File: src/index.js (hunk 3 above)
Comment (srishtigautam21) [reply src/index.js:31]: one is CART and other CARD provider

File: src/component/Context/PriceContext/PriceContext.js (hunk 2 above)
Comment (srishtigautam21) [reply src/component/Context/PriceContext/PriceContext.js:1]: Thanks Ashish

**Review by Muzammil-khan710 — APPROVED** [review]
Everything looks great here..
Good to merge !

**Pull request merged**